src/component/AddItem.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from "react";
+import { withFirestore } from "react-firestore";
+
+const AddItem = ({ firestore }) => {
+  const [name, setName] = useState("");
+
+  //   Write item to Firebase
+  const addItem = name => {
+    firestore.collection("items").add({ name });
+  };
+
+  //   Update state whenever user input changes
+  const handleChange = event => {
+    setName(event.target.value);
+  };
+
+  //   Trigger addItem function when "Add Item" button is clicked
+  const handleSubmit = event => {
+    event.preventDefault();
+    addItem(name);
+    setName("");
+    // alert("item added to database!");
+  };
+
+  return (
+    <div>
```

Comment:
Technically, you don't need this `<div>` because the `<form>` tag contains all of the other tags on the page. In practice, this page will probably grow up to have other stuff on it, so you'll likely want some kind of container. What that is might depend on what you do with my comment in `App.js` ... Leaving this a little ambiguous in the interest of time. Let me know if you want clarification!